Sources/Plasma/Apps/plClient/external/render_svg.py

```diff
@@ -247,7 +248,12 @@ def render_voice_icons(inpath, outpath):
 	outpath = os.path.expanduser(options.outpath)
 	inpath = os.path.expanduser(options.inpath)
 
-	if not os.path.exists(outpath):
+	# Ensure that the render directory is empty, preventing old renders from junking the output
+	if os.path.exists(outpath):
+		print("Cleaning render directory...")
+		for i in glob(os.path.join(outpath, "*.png")):
+			os.unlink(os.path.join(outpath, i))
```

Comment:
I'm slightly uncomfortable with an automated tool doing wildcard unlinks, but this looks relatively safe.